src/db-service.ts

```diff
@@ -250,4 +272,26 @@ export class ItemValidationService {
       )
       .then(({ rows }) => rows[0]);
   }
+
+    /**
+   * Toggle the enabled field of an item validation process
+   * @param id process id
+   * @param {boolean}
+   */
+     async toggleEnabledForItemValidationProcess(
```

Comment:
Actually (sorry if I made you change it before..) I think this should be `setEnabledForItemValidationProcess` since your just applying sql.json (and not actually inverting it)